lib/parameters.js

```diff
@@ -95,8 +95,8 @@ function getFullFileList(peliasConfig, args) {
   const files = _.get(peliasConfig.imports.openaddresses, 'files', []);
 
   if (_.isEmpty(files)) {
-    // no specific files listed, so return all .csv files
-    return glob.sync( args.dirPath + '/**/*.csv' );
+    // no specific files listed, so return all .csv and .geojson files
+    return glob.sync( args.dirPath + '/**/*.csv' ).concat(glob.sync( args.dirPath + '/**/*.geojson' ));
```

Comment:
I didn't test it out, but many glob libraries support 'braces':
```
/**/*.{csv,geojson}
```

Not a big deal, but might save a little disk spinning 😄

Reply:
yes, this work like a charm :smile: